VideoLocker/res/layout-land/player_controller.xml

```diff
@@ -1,242 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The landscape layout has some different padding/margin dimensions defined than the base layout (e.g. `lms_btn` has top/bottom padding). Since we need to merge these into one layout due to our current player implementation not supporting layout changes upon orientation change, I hope we can use some reasonable values that look good on both…